[DevNet] tweaks for remote environments (#601)
pokt-network/pocket: docs/releases/README.md

## Container Images

Our images are hosted on GitHub’s Container Registry (GHCR) and are available at `ghcr.io/poktnetwork/pocket-v1`. You can find the list of latest images [here](https://github.com/pokt-network/pocket/pkgs/container/pocket-v1).

### Tags

Code built from the default branch (i.e., `main`) is tagged as `latest`.

> Code built from commits in Pull Requests, is tagged as `pr-<number>`, as well as `sha-<7 digit sha>`. If you need to build an image from a Pull Request, attach a `push-image` label to the pull request to force CI to upload a container image from a non-main branch.

Reviewer: ```suggestion
Code built from commits in Pull Requests, is tagged as `pr-<number>`, as well as `sha-<7 digit sha>`. You can optionally build an image from a Pull Request by attaching a `push-image` label to the pull request to force CI to upload a container image from a non-main branch.
```

> Code built from commits in Pull Requests, is tagged as `pr-<number>`, as well as `sha-<7 digit sha>`. You can optionally build an image from a Pull Request by attaching a `push-image` label to the pull request to force CI to upload a container image from a non-main branch.

Reviewer 2: Should protocol-infra docs reference this convention?

Author: @dylanlott Haha, they do: https://github.com/pokt-network/protocol-infra#how-to-run-a-pocket-version-from-the-developers-branch
